app/src/main/java/net/skyscanner/backpack/demo/compose/BpkStarRatingStory.kt

```diff
@@ -0,0 +1,127 @@
+ *
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import net.skyscanner.backpack.compose.starrating.BpkHotelRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRatingSize
+import net.skyscanner.backpack.compose.starrating.RoundingType
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun BpkStarRatingStory(
+  modifier: Modifier = Modifier,
+) {
+  Box(
+    modifier = modifier.fillMaxHeight(),
+    contentAlignment = Alignment.Center,
+  ) {
+    Column(modifier = Modifier.padding(start = BpkSpacing.Lg)) {
+      BpkText(
+        text = stringResource(R.string.static_default_rating_title),
+        style = BpkTheme.typography.heading2,
+      )
+      Row() {
+        Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+          StaticRatingSample(
+            size = BpkStarRatingSize.Large,
+            text = stringResource(R.string.icons_large),
+          )
+        }
+        Column() {
+          StaticRatingSample(
+            size = BpkStarRatingSize.Small,
+            text = stringResource(R.string.icons_small),
+          )
+        }
+      }
+      Spacer(modifier = Modifier.padding(top = BpkSpacing.Lg))
+      Column {
+        BpkText(
+          text = stringResource(R.string.hotel_star_rating_title),
+          style = BpkTheme.typography.heading2,
+        )
+        Row {
+          Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+            HotelRatingSample(
+              size = BpkStarRatingSize.Large,
+              text = stringResource(R.string.icons_large),
+            )
+          }
+          Column {
+            HotelRatingSample(
+              size = BpkStarRatingSize.Small,
+              text = stringResource(R.string.icons_small),
+            )
+          }
+        }
+      }
+    }
+  }
+}
+
+@Composable
+private fun HotelRatingSample(size: BpkStarRatingSize, text: String) {
+  BpkText(
+    text = text,
+    style = BpkTheme.typography.heading3,
+  )
+  for (i in 1..5) {
+    BpkHotelRating(
+      rating = i,
+      contentDescription = { value, max ->
+        stringResource(R.string.star_rating_accessibility_status, value, max)
+      },
+      size = size,
+    )
+  }
+}
+
+@Composable
+private fun StaticRatingSample(size: BpkStarRatingSize, text: String) {
```

Comment:
Instead of wrapping each call of this function in Column, please use Column here inside (same can be allied to HotelRatingSample). Also, please provide modifiers to these functions